lib/bolt/node/winrm.rb

```diff
@@ -220,12 +221,18 @@ def process_from_extension(path)
     end
 
     def _upload(source, destination)
+      write_remote_file(source, destination)
+      Bolt::Result.new
+    # TODO: we should rely on the executor for this
+    rescue StandardError => ex
+      Bolt::Result.from_exception(ex)
+    end
+
+    def write_remote_file(source, destination)
       @logger.debug { "Uploading #{source} to #{destination}" }
       fs = ::WinRM::FS::FileManager.new(@connection)
+      # TODO: raise FileError here if this fails
```

Comment:
That seems simple, still planning to add it?

Reply:
I'm not sure what the failure cases look like or what message to generate